modules/ROOT/Edits/configure-third-party.adoc

```diff
@@ -0,0 +1,21 @@
+= Configure and Manage 3rd Party Connections
```

Comment:
GLOBALLY spell out ordinals: in this case, "third-party <noun object>"
```suggestion
= Configure and Manage Third-Party Connections
```
I thought this was in the style guide, but it's not, so I will add it: thanks!